src/components/TxDetailsModal.js

```diff
@@ -29,11 +30,21 @@ class TxDetailsModal extends Component {
   });
 
   render() {
+    /**
+     * @type {{
+     *   token: unknown;
+     *   tx: TxHistory;
+     *   isNFT: boolean;
+     * }} TxDetailsModal properties
+     */
     const { token, tx, isNFT } = this.props;
     const fullTokenStr = getTokenLabel(token);
     const description = tx.getDescription(token);
     const timestampStr = tx.getTimestampFormat();
     const idStr = getShortHash(tx.txId, 12);
+    const ncId = tx.ncId && getShortHash(tx.ncId, 7);
+    const ncMethod = tx.ncMethod;
+    const ncCallerAddress = tx.ncCaller && getShortContent(tx.ncCaller.base58, 7);
```

Comment:
@pedroferreira1 That is the motivation to get nano contract basic information in `IHistoryTx` as I pointed at this [comment](https://github.com/HathorNetwork/hathor-wallet-lib/pull/613#discussion_r1513496110).